### ISSUE TITLE
Resume: Remove vertical decorator for specific section 

### ISSUE THREAD

**Issue by abruegger** [nstrayer/datadrivencv#80]
I added a new section with publications that doesn't require a vertical bar at the left (used for dates in "normal" sections). I managed to change this section in some regards (e.g., placing text at desired place on page with
). But I can't figure out how to remove the bar, which is distracting here.

![remove_decorator](https://user-images.githubusercontent.com/31000762/147818976-5c97b106-92b7-425d-9c37-16a12a90262e.png)

Is there a way to remove the vertical decorator line after the icon in Resume? I tried to turn it off via a custom CSS file or via a code snippet within the .Rmd file but so far without success. I guess my main problem is that I don't know how to identify / access the element that creates the bar.

Please note that I would like to keep the line for the other sections for which it is useful (e.g., positions).

**Comment by edwardhealey** [comment]
did you have any luck with this? I would like to edit this line too but not sure where it is located in the package

**Comment by abruegger** [comment]
Yes, see here: https://github.com/rstudio/pagedown/issues/260#issuecomment-1125845862
